tests/Unit/MSFT_xDnsRecordMx.Tests.ps1

```diff
@@ -0,0 +1,157 @@
+$script:dscModuleName = 'xDnsServer'
+$script:dscResourceName = 'MSFT_xDnsRecordMx'
+
+function Invoke-TestSetup
+{
+    try
+    {
+        Import-Module -Name DscResource.Test -Force -ErrorAction 'Stop'
+    }
+    catch [System.IO.FileNotFoundException]
+    {
+        throw 'DscResource.Test module dependency not found. Please run ".\build.ps1 -Tasks build" first.'
+    }
+
+    $script:testEnvironment = Initialize-TestEnvironment `
+        -DSCModuleName $script:dscModuleName `
+        -DSCResourceName $script:dscResourceName `
+        -ResourceType 'Mof' `
+        -TestType 'Unit'
+
+    Import-Module (Join-Path -Path $PSScriptRoot -ChildPath 'Stubs\DnsServer.psm1') -Force
+}
+
+function Invoke-TestCleanup
+{
+    Restore-TestEnvironment -TestEnvironment $script:testEnvironment
+}
+
+Invoke-TestSetup
+
+try
+{
+    InModuleScope $script:dscResourceName {
+        #region Pester Test Initialization
+        $dnsRecordsToTest = @(
+            @{
+                TestParameters = @{
+                    Name      = '.'
+                    Zone      = 'contoso.com'
+                    Target    = 'mail.contoso.com'
+                    Priority  = 20
+                    TTL       = '02:00:00'
```

Comment:
You don't have any tests for if these parameters are not used, Priority and TTL.